template.tpl

```diff
@@ -767,104 +759,217 @@ ___TEMPLATE_PARAMETERS___
         "newRowButtonText": "Add configuration"
       },
       {
-        "enablingConditions": [
-          {
-            "paramName": "attribution",
-            "type": "EQUALS",
-            "paramValue": true
-          }
-        ],
-        "displayName": "Attribution Options",
-        "name": "attributionOptions",
-        "groupStyle": "ZIPPY_OPEN",
-        "type": "GROUP",
+        "help": "Wether to use client side user agent enrichment. Enable client side user agent to make the changes compatible with previous GTM template version.",
+        "defaultValue": false,
```

Comment:
I also want to make client UA parsering disabled by default as it will make all the latest SDKs have the same UA logic.

However, since this is a breaking change, that may be automatically applied to customers I think we should go our of our way to call this out in a public announcement. In addition to any migration docs/update notes in the developer center I think we should also create a community post that summarizes the changes in this release, and how customers can "opt-in" to the old functionality if they want.

Reply:
Sync offline. 
We will list the breaking changes in the doc center, post a community post to point out the breaking changes, and also add the notes in change note or template discription.